crates/bevy_render/src/mesh/mesh.rs

```diff
@@ -411,6 +411,15 @@ pub mod shape {
 
     impl From<Icosphere> for Mesh {
         fn from(sphere: Icosphere) -> Self {
+            if sphere.subdivisions >= 80 {
+                let temp_sphere = Hexasphere::new(sphere.subdivisions, |_| ());
+
+                panic!(
+                    "Cannot create an icosphere of {} subdivisions due to there being more than 2¹⁶ vertices being generated: {} (Limited to 65535 vertices or 79 subdivisions)",
```

Comment:
I don't know if that message using the unicode `¹⁶` is potentially a compatability hazard. It might be worth saying `2¹⁶ (65536)`. Also I imagine we can only actually have 65535 vertices anyway.

Reply:
I don't think it's necessary to include the `2^16` anyway. I think that just sticking to the extra note at the end which remarks there being a limit is the correct call.